tests/test_new_window_handle.py

```diff
@@ -0,0 +1,17 @@
+from foxpuppet import FoxPuppet
+from time import sleep
+
+
+class Test_new_window_handle:
+
+    def test_new_private_window(self, selenium):
+
+        foxpuppet = FoxPuppet(selenium)
+        selenium.get('http://www.google.com')
```

Comment:
Please stop writing tests that depend on Google or any other external website. 😄